torch_tvm/fusion_pass.cpp

```diff
@@ -24,7 +24,7 @@ value_list sortReverseTopological(ArrayRef<Value*> inputs, Block* block) {
 bool canHandle(Block* block, AliasDb& aliasDb);
 bool canHandle(Node* node, AliasDb& aliasDb) {
   if (node->kind() == prim::Constant) {
-    return true;
+    return false;
```

Comment:
Why this?

Reply:
Aah I thought I removed this. Let me double check. Good catch.

Reply:
OK, had forgotten push the latest.